Added download file method to virtual machine

Steven17D opened the pull request.

Commit:
Added download file method to virtual machine

jossef commented on vmwc/__init__.py:
```diff
@@ -663,6 +663,27 @@ def send_file(self, source_path, guest_path):
+
+        size = 0
+        with open(local_path, 'wb') as f:
+            for chunk in resp.iter_content(chunk_size=1024):
```
the `1024` can be placed as an optional parameter so the user may change it when required;

```
def download_file(self, guest_path, local_path, chunk_size=1024)
```

jossef commented on vmwc/__init__.py:
```diff
@@ -663,6 +663,27 @@ def send_file(self, source_path, guest_path):
+                                                                                                    auth=self._tools_credentials,
+                                                                                                    guestFilePath=guest_path)
+
+        resp = requests.get(fti.url.replace("https://*", "https://" + self._client.host), verify=False)
```
you can use `self._client.verify` to determine if SSL verification is required in this session

jossef commented on examples/virtual-machine-download-file.py:
```diff
@@ -0,0 +1,24 @@
+                'Failed to download file: {}'.format(e)
+
+if __name__ == '__main__':
+    main()
```
great example, add it to the `README.md` file as well 
😉

jossef reviewed (changes requested):
:1st_place_medal:

Commit:
Added chunk_size as optional argument to download_file method.
Make use of client's verify attribute.
Added download_file example to README.md

Steven17D commented:
Hi Jossef, I fixed all the notes that you gave me.
